webperf-metrics/compare/index.js

```diff
@@ -0,0 +1,20 @@
+const core = require('@actions/core');
+const path = require('path');
+
+const { createMetricsComparison } = require('./lib/create-metrics-comparison');
+const getArg = require('../../libs/utils/get-arg');
+
+const cwd = process.cwd();
+const baseBranchMetricsFilePath = path.join(cwd, getArg('baseBranchMetricsPath'));
```

Comment:
```suggestion
const baseBranchMetricsFilePath = path.join(cwd, getArg('baseBranchMetricsPath'), 'metrics.json');
```
zmienna `baseBranchMetricsPath` zawiera tylko nazwę katalogu aktualnie, https://github.com/allegro-internal/opbox-component-template/blob/a7ec393df80c0fa3ddb36d1d9ee1d5f8a60abee5/.github/workflows/webperf-check.yml#L7, czyli żebyś miał tutaj ścieżkę to musisz jeszcze znać nazwę pliku

Reply:
dzięki, done